apps/labs/posts/three-months-and-having-fun.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Quansight Labs: what I learned in my first 3 months'
+published: July 21, 2020
+author: matti-picus
+description: 'I joined Quansight at the beginning of April, splitting my time between
+PyTorch (as part of a larger Quansight team) and contributing to Quansight Labs
+supported community-driven projects in the Python scientific and data science
+software stack, primarily to NumPy.  I have found my next home; the people, the
+projects, and the atmosphere are an all around win-win for me and (I hope) for
+the projects to which I contribute.'
+category: [PyData Ecosystem]
+featuredImage:
+  src: /posts/three-months-and-having-fun/blog_feature_org.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/three-months-and-having-fun/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+I joined Quansight at the beginning of April, splitting my time between
+PyTorch (as part of a larger Quansight team) and contributing to Quansight Labs
+supported community-driven projects in the Python scientific and data science
+software stack, primarily to NumPy.  I have found my next home; the people, the
+projects, and the atmosphere are an all around win-win for me and (I hope) for
+the projects to which I contribute.
+
+I am not a newcomer to Open Source. I originally became involved in
+[PyPy](https://www.pypy.org) as an after-hours hobby to hone my developer
+skills, and quickly became enamoured with the people and the mission. Over the
+years my efforts in the open source world moved more mainstream, and in 2018 I
+took on a full-time position working on NumPy, funded through a grant to
+[BIDS](https://bids.berkeley.edu). Since April 2020, I have moved to Quansight
+Labs as a full-time developer. 
+
+[Quansight Labs](https://www.quansight.com/labs) is a subsidiary of Quansight
+LLC, and Labs' mission is "Sustaining the future of open source". It does this
+by gathering together an amazing group of software developers and letting them
+loose on a variety of open source projects: Numpy, Spyder, Jupyter, conda-forge,
+Numba and [more](http://labs.quansight.org/categories). Of course, the
```

Comment:
@mattip since our new blog doesn'T have a tags page, would you like me to replace this link with the suggested blog link, or with a link to the [new projects page](https://labs.quansight.org/projects).)?
```suggestion
Numba and [more](https://labs.quansight.org/blog). Of course, the
```